assets/css/style.css

```diff
@@ -0,0 +1,11 @@
+*
+{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+h1
+{
+    color: blue;
```

Comment:
Color could be red maybe. What do you think?

Reply:
I think orange is okay :D